test/e2e/resize_nodes.go

```diff
@@ -358,12 +393,15 @@ var _ = Describe("Nodes", func() {
 			err := waitForPodsCreatedRunningResponding(c, ns, name, replicas)
 			Expect(err).NotTo(HaveOccurred())
 
-			By("cause network partition on one node")
-			nodelist, err := c.Nodes().List(labels.Everything(), fields.Everything())
+			By("chose node with at least one pod - the one will fail")
```

Comment:
Improved:
By("choose a node with at least one pod - we will block network traffic on this node")